### PR TITLE
Updated Sass version to 1.61.0

### PR DESCRIPTION
**Pull request by azizk**
The Linux package structure has changed and also has two binaries in the `src/` folder now. `dart-sass` 1.57.1 was the last version that you could configure for `:dart_sass` on Linux.

With my changes you can use newer versions, but you won't be able to use earlier versions anymore.

I refactored the code somewhat for improvements and better readability. I'll revert specific changes if you don't like some. :slightly_smiling_face:

### CODE REVIEW

**Comment by mcrumm** [comment]
Hi @azizk, thanks for the PR! Can you please revert any changes other than those required to get dart_sass running on Linux again? That will make it much easier to review, thanks! If it's easier to close this PR and open a new one, that is fine too :)

**Comment by azizk** [comment]
Sure, np! 😀

**Comment by azizk** [comment]
Hey @mcrumm, I extracted the unrelated refactorings from this branch and I'll make a different PR for those. Hope it's easier to review now. :slightly_smiling_face:

**Comment by azizk** [comment]
Hi @mcrumm, hope you're not too busy. Any chance this can be merged in today or tomorrow? Thanks! :smiley:

File: lib/dart_sass.ex [hunk 1]
```diff
@@ -183,21 +169,7 @@ defmodule DartSass do
...
-      if "--watch" in args and platform() != :windows do
-        [script_path() | path]
-      else
-        path
-      end
-
-    path
-    |> cmd(args, opts)
-    |> elem(1)
+    run_cmd(bin_paths(), config_args ++ extra_args, opts) |> elem(1)
```
Comment (mcrumm) [inline lib/dart_sass.ex:172]: We still need `script_path()` unfortunately. :( The upstream PR was merged but the changes had to be reverted due to issues on Windows.

File: priv/dart_sass.bash
Comment (mcrumm) [inline priv/dart_sass.bash:1]: This script is still required for `--watch`.

**Review by mcrumm — COMMENTED** [review]
Thanks @azizk! Just one small change and we can :shipit:

File: lib/dart_sass.ex (hunk 1 above)
Comment (azizk) [reply lib/dart_sass.ex]: Ah, that's a pity, but okay. Done!

**Comment by mcrumm** [comment]
🍬 🍬 🍬 🍬 🍬

**Pull request merged**